### PR TITLE
Adjustments to Accordion component:

### PR DESCRIPTION
**Pull request by paboden**
- Added hover stying to label/summaries
- Adjusted padding sizes to account for hover state design.

### CODE REVIEW

File: src/stories/Molecules/Accordion/Accordion.css [hunk 1]
```diff
@@ -48,6 +52,15 @@
   transform:rotate(225deg);
 }
 
+.accordion details > summary:hover {
+  background-color: var(--color-base-light-xxx);
+  color: var(--accent-color);
+}
+
+.accordion.stack > * + * {
```
Comment (sleepingmonk) [inline src/stories/Molecules/Accordion/Accordion.css:60]: You appear to have removed `stack` from the classes in twig, so I think this code doesn't do anything.

File: src/stories/Molecules/Accordion/Accordion.twig [hunk 2]
```diff
@@ -6,14 +6,15 @@
 {# Include variant classes form above. #}
 {% set classes = [
   'accordion',
-  'stack',
   variant_class,
 ] %}
 
 <div {{ attributes.addClass(classes) }}>
```
Comment (sleepingmonk) [inline src/stories/Molecules/Accordion/Accordion.twig:12]: Might as well clean up attributes from all Accordion files while you're in here.

Comment (paboden) [reply src/stories/Molecules/Accordion/Accordion.twig]: Cleaned up :)

File: src/stories/Molecules/Accordion/Accordion.css (hunk 1 above)
Comment (paboden) [reply src/stories/Molecules/Accordion/Accordion.css]: removed :)

**Comment by sleepingmonk** [comment]
Merged locally after rebase.

**Pull request closed without merging**